trade-finance-manager-ui/server/controllers/case/cancellation/check-details.controller.ts

```diff
@@ -79,6 +79,10 @@ export const postDealCancellationDetails = async (req: PostDealCancellationDetai
 
     await api.submitDealCancellation(_id, { reason, bankRequestDate, effectiveFrom }, userToken);
 
+    const { ukefDealId } = deal.dealSnapshot.details;
+
+    req.flash('successMessage', `Deal ${ukefDealId} cancelled`);
```

Comment:
What does `api.submitDealCancellation` return please? I'm wondering if `api.submitDealCancellation` fails, does the flash still occur?

Reply:
It throws an error, which means here it will skip the `req.flash` section of code and go straight to the error handling block in this file

Reply:
Brilliant thanks @BethThomas141